.gitignore

```diff
@@ -1,9 +1,11 @@
 node_modules
-yarn.lock
```

Comment:
Is there a specific reason we want to include `yarn.lock`? I'm just not too familiar with yarn...

Reply:
Yep, the [recommendation](https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/) is to commit this to source control